### DISCUSSION TITLE
`AnimatedPoint` and `AnimatedLineOffsets` do not have a `<Page />` around them

### DISCUSSION THREAD

**Discussion by naftalibeder** [rnmapbox/maps#3454] [General]
This means there is no way to go back to the examples list from them.

What was the intention with this? I remember @mfazekas asking to remove the `<Page />` wrapper but I don't remember why or whether I did.

**Comment by mfazekas** [comment]
@naftalibeder we're moving away from `Page` in examples, the idea is that if possible example should be something you can copy and paste into your app. Pages are added automatically to examples. But Animation is not yet converted to this.  It's my laziness sorry for the inconsistency.


https://github.com/rnmapbox/maps/blob/b76c000a237b9757a616982d6c07f6ecfd7d60a9/example/src/scenes/GroupAndItem.tsx#L258-L261

https://github.com/rnmapbox/maps/blob/b76c000a237b9757a616982d6c07f6ecfd7d60a9/example/src/scenes/GroupAndItem.tsx#L298-L310

**Reply by naftalibeder** [reply]
No worries, thanks for the update :)
